zkvm/docs/zkvm-spec.md

```diff
@@ -828,6 +787,53 @@ V == v·B + 0·B2
 1. Prover shows `v`.
 2. Verifier checks equality `V == v·B`.
 
+### Taproot
+
+Taproot provides efficient and privacy-preserving storage of smart contracts. It is based on [a proposal by Gregory Maxwell](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2018-January/015614.html). Multi-party blockchain contracts typically have a top-level "success clause" (all parties agree and sign) or "alternative clauses" to let a party exit the contract based on pre-determined constraints. Our design of Taproot has three significant features.
```

Comment:
>Our design of Taproot has three significant features.

Unclear what this means. I think we are pretty much implementing Taproot as-is at this point, without major changes. Hashing is different, but that does not affect the shape of the scheme.